src/dqe.erl

```diff
@@ -324,33 +368,25 @@ name({named, N, Q}, Aliases, Buckets) ->
             {ok, {N, Q1}};
         E ->
             E
-    end;
-
-name(Q, Aliases, Buckets) ->
-    case translate(Q, Aliases, Buckets) of
-        {ok, Q1} ->
-            {ok, {dql:unparse(Q), Q1}};
-        E ->
-            E
     end.
 
-keep_optimizing_mget([_, _, _, _, _ | _] = Gets) ->
-    keep_optimizing_mget(optimize_mget(Gets));
-keep_optimizing_mget(Gets) ->
+keep_optimizing_sum([_, _, _, _, _ | _] = Gets) ->
+    keep_optimizing_sum(optimize_sum(Gets));
+keep_optimizing_sum(Gets) ->
     Gets.
 
-optimize_mget([G1, G2, G3, G4]) ->
-    [{dqe_mget, [[G1, G2, G3, G4]]}];
+optimize_sum([G1, G2, G3, G4]) ->
+    [{dqe_sum, [[G1, G2, G3, G4]]}];
```

Comment:
The code in the following (LINE, COL) locations has the same structure: (379, 6), (382, 6).